packages/user-mgnt/src/features/system/handler.ts

```diff
@@ -302,13 +304,11 @@ export async function getSystemHandler(
     throw unauthorized()
   }
 
-  const systemName = `${system.name[0]?.given || ''} ${
-    system.name[0]?.family || ''
-  }`.trim()
-  const createdBy = `${system.createdBy[0]?.given} ${system.createdBy[0]?.family}`
+  const systemName = system.name
+  const createdBy = system.createdBy
   return {
     name: systemName || createdBy,
-    createdBy: `${system.createdBy[0]?.given} ${system.createdBy[0]?.family}`,
+    createdBy: createdBy,
```

Comment:
nitpick: maybe we could just use "createdBy: system.createdBy" here to avoid the unnecessary repetition.